regression/test.pl

```diff
@@ -227,6 +228,8 @@ ($$$$)
   -T         thorough: run expensive tests
   -F         future: run checks for future features
   -K         known: run tests associated with known bugs
+  -P         platform: list of arguments which replaces the \$PLATFORM string
+             in .desc files
```

Comment:
What is the difference between `test.pl -c cbmc -P bla` and `test.pl -c "cbmc bla"` ?

Reply:
The former is only applied to specific tests, the latter to all tests.
I don't want to break any existing tests, I just want to provide an option to create new tests with standardised parameter set.

Reply:
Indeed I was wrong to conflate the two approaches; what I continue to struggle with is 1) the name `PLATFORM` and 2) how to document what good, common values are. My assumption would be that over time you will want a `$PLATFORM1`, `$PLATFORM2`, etc.

Reply:
I honestly struggle to find a better name. "defaults" or "common" would be misleading, as one would assume that those arguments would be added to all tests. Merged with existing ones.

We don't foresee adding extra `$PLATFORM`s, as there's a singular "platform" we're using.

Reply:
I am running the risk of being very narrow-minded, hence the following is a serious question: isn't a "platform" more commonly an operating system, a particular hardware, or the combination of those? It might be just about changing this to `PLATFORM_OPTS` for clarity.

If you are talking about a single `$PLATFORM`, are just really just introducing a shorthand to save some typing?